cylc/flow/suite_files.py

```diff
@@ -429,15 +429,13 @@ def get_workflow_source_dir(
             return None, None
 
 
-def get_suite_srv_dir(reg, suite_owner=None):
+def get_suite_srv_dir(reg):
     """Return service directory of a suite."""
-    if not suite_owner:
-        suite_owner = get_user()
     run_d = os.getenv("CYLC_SUITE_RUN_DIR")
     if (
         not run_d
         or os.getenv("CYLC_SUITE_NAME") != reg
-        or os.getenv("CYLC_SUITE_OWNER") != suite_owner
+        or os.getenv("CYLC_SUITE_OWNER") != get_user()
```

Comment:
If `suite_owner` was always `None` before I think we can remove it?

```suggestion
```

Reply:
No probs, it defaulted to get_user() if it was none, just for purposes of documenting in the contact file, but I can remove the whole field if it is not a useful one.